### PR TITLE
Dev/juan

### PR DESCRIPTION
**Pull request by juanc37**
Fixed recent issues

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -53,8 +53,19 @@ function readFileCSV(text, fileName) {
...
+            text: "Please only drag and drop CSV files given to you from the Blackboard Grade Center. We were not able to detect all of the columns necessary to convert your file.",
+            icon: "warning",
+            buttons: true,
+            dangerMode: true
+        });
+        return;
+    }
     for (var j = 1; j < allTextLines.length - 1; j++) { //skip header line
         line = allTextLines[j].split(',');
+        if (line[user_name].substr(0,8) === "\"clicker") continue; //skip over clicker users
```
Comment (tpaulus) [inline app.js:68]: Can we also include Preview Users in the skip list?

**Review by tpaulus — CHANGES_REQUESTED** [review]

File: app.js (hunk 1 above)
Comment (juanc37) [reply app.js:68]: updated in my branch, we just need to test

**Review by tpaulus — APPROVED** [review]
Approved - Provided it is tested before merging.

**Review by tpaulus — APPROVED** [review]

**Comment by juanc37** [comment]
Need to restructure code per toms request

**Pull request merged**